mcstatus/motd.py

```diff
@@ -0,0 +1,280 @@
+from __future__ import annotations
+
+import re
+from dataclasses import dataclass
+from enum import Enum
+from typing import List, Optional, TYPE_CHECKING, Tuple, Union
+
+if TYPE_CHECKING:  # pragma: no cover
```

Comment:
Even though this is indeed something which shouldn't be reported in coverage reports, we didn't ignore these conditions anywhere else in the code, and there's a better way to address this by changing the `.coveragerc` file, which I will do in another PR. Good catch, but it doesn't belong here.

```suggestion
if TYPE_CHECKING:
```

Reply:
Removed comment and merge master to this PR [here](https://github.com/py-mine/mcstatus/compare/bccee991168674292f251d33cc2c2f19584abfb9..b09a9e2fd06e0109fd3a25b7357725f1e80d9034).